GNUmakefile

```diff
@@ -85,6 +85,16 @@ go-solidity-wrappers-ocr2vrf: pnpmdep abigen ## Recompiles solidity contracts an
 generate: abigen codecgen mockery ## Execute all go:generate commands.
 	go generate -x ./...
 
+.PHONY: testscripts
+testscripts: ## Install and run testscript against testdata/scripts/* files.
+	go install github.com/rogpeppe/go-internal/cmd/testscript@latest
+	go install -ldflags "-X github.com/smartcontractkit/chainlink/v2/core/static.Version=unset -X github.com/smartcontractkit/chainlink/v2/core/static.Sha=unset"
```

Comment:
This works, but changing the local installation permanently is not ideal.
I tried to just `go build` in the working dir and target via `PATH` or `alias`, but then I get:
> [exec: "chainlink": cannot run executable found relative to current directory]

Reply:
Fixed with `$(CURDIR)` instead of `.`.